docs/tilt.md

```diff
@@ -0,0 +1,30 @@
+# Rapid iterative development using Tilt
+
+## Requirements
+
+- [Tilt](https://tilt.dev/)
+- kubectl
+
+## Workflow
+
+Set your environment:
+
+```shell
+# Name of your cluster's k8s context (check kubectl config get-contexts).
+export IG_K8S_CLUSTER=my-cluster
+# A Docker registry to use for development. Tilt will push temporary images there.
+export IG_REGISTRY=quay.io
+# A Docker repository to use for development. Tilt will push temporary images there.
+export IG_REPOSITORY=me/inspektor-gadget
+```
```

Comment:
In the Makefile and in the CI, we use this variable:
```
CONTAINER_REPO ?= ghcr.io/kinvolk/inspektor-gadget
```

Could we reuse the same convention?

Reply:
Done.